Week_6/sentimental-cash/cash.py

```diff
@@ -0,0 +1,30 @@
+def main():
+    try:
+      change = int(input("How much change is owed: "))
```

Comment:
```suggestion
       change = int(input("How much change is owed: "))
```
Check indentation here and in line 5-6